docs/querying.md

```diff
@@ -38,6 +38,18 @@ With direction
 Post.order(created_at: :desc, title: :asc)
 ```
 
+## Group
+
+Order is using the QueryBuilder and supports providing an GROUP BY clause:
+```crystal
+posts = Post.group(:published)
```

Comment:
I think I prefer `group_by` instead.  wdyt?

Reply:
me too prefer `group_by` and `order_by` over `order`
it was `order` so i did `group`